pkg/cmd/pr/view/view.go

```diff
@@ -272,7 +274,8 @@ func formattedReviewerState(cs *iostreams.ColorScheme, reviewer *reviewerState)
 		colorFunc = func(str string) string { return str } // Do nothing
 	}
 
-	return fmt.Sprintf("%s (%s)", reviewer.Name, colorFunc(strings.ReplaceAll(strings.Title(strings.ToLower(state)), "_", " ")))
+	c := cases.Title(language.English)
+	return fmt.Sprintf("%s (%s)", reviewer.Name, colorFunc(strings.ReplaceAll(c.String(strings.ToLower(state)), "_", " ")))
```

Comment:
I would actually be in favor of a switch statement here listing all known machine identifiers for reviewer state (PENDING, COMMENTED, APPROVED, CHANGES_REQUESTED, DISMISSED) and translating them to human-readable labels. I think it would eliminate the complex logic here and the guessing of what values will actually be rendered